app/database/operator/server_data_operator/handlers/post.ts

```diff
@@ -31,6 +31,8 @@ const {
     POST,
 } = MM_TABLES.SERVER;
 
+type PostActionType = keyof typeof ActionType.POSTS;
```

Comment:
Why you need `keyof typeof` insteado of just `keyof`?

Reply:
because I want it to match the string value of "RECEIVED_IN_CHANNEL" | "RECEIVED_IN_THREAD" | "RECEIVED_SINCE" | "RECEIVED_AFTER" | "RECEIVED_BEFORE" |...